source/SharpGL/Core/SharpGL/RenderContextProviders/DIBSectionRenderContextProvider.cs

```diff
@@ -84,15 +84,18 @@ public override void SetDimensions(int width, int height)
 
 	    public override void Blit(IntPtr hdc) 
 	    {
+            // [RS] Because of this warning, we remove the if-statement.
+            // Warning 1: The result of the expression is always 'true' since a value of type 'System.IntPtr' is never equal to 'null' of type 'System.IntPtr?'	F:\Git Repositories\SharpGL\source\SharpGL\Core\SharpGL\RenderContextProviders\DIBSectionRenderContextProvider.cs	88	17	SharpGL
+
             //  We must have a device context.
-            if (deviceContextHandle != null)

```

Comment:
Perfect! I like this style! :) So I'll do that and send it via PR targeted to 2.4.